test/toolshed/unix_test.exs

```diff
@@ -10,4 +11,9 @@ defmodule Toolshed.UnixTest do
     {date_time, 0} = System.cmd("date", ["-u"], env: %{"LC_ALL" => "C"})
     String.trim(date_time)
   end
+
+  test "cat/1 reads file content" do
+    assert capture_io(fn -> Unix.cat("test/support/test_file.doc") end) =~
+             "Content of this will be read for test purposes\n"
```

Comment:
I'd change this to be a strict `==` rather than the regex `=~` since it seems like `cat` shouldn't modify its output.

```suggestion
    assert capture_io(fn -> Unix.cat("test/support/test_file.doc") end) ==
             "Content of this will be read for test purposes"
```

I removed the `\n` from the end of the line, since `cat` shouldn't be adding characters to the output. That means that there's a bug in the implementation. The fix would be to use `IO.write/2` which won't add the newline.